app/serializers/participant_serializer.rb

```diff
@@ -30,10 +30,8 @@ class ParticipantSerializer
   end
 
   attributes :participant_type do |user|
-    if user.early_career_teacher_profile
-      :ect
-    else
-      :mentor
+    if participant_active?(user)
+      user.early_career_teacher? ? :ect : :mentor
```

Comment:
I wonder if for our sanity's sake we should make it something like
```
if user.early_career_teacher?
  :ect
elsif user.mentor?
  :mentor
end
```
This should still return `nil` now that you added the active restriction to mentor and ect profiles on the user, and should return `nil` for npq users too, if somehow they end up here. As opposed to returning `:mentor` for npq users.

Reply:
Yep, that sounds more robust